src/http-driver.js

```diff
@@ -103,14 +108,12 @@ function makeHTTPDriver({eager = false} = {eager: false}) {
       .map(reqOptions => {
         let response$ = createResponse$(reqOptions)
         if (eager || reqOptions.eager) {
-          response$ = response$.replay(null, 1)
-          response$.connect()
+          response$ = response$.publishReplay(null, 1).connect()
+          return response$
         }
         response$.request = reqOptions
         return response$
-      })
-      .replay(null, 1)
-    response$$.connect()
+      }).shareReplay(null, 1)
```

Comment:
So response$$ defined with `publishReplay(1)` + `connect()` does **not** work? You sure?
And by the way, the initial value given to publishBehavior() for `response$` should **not** be the empty Observable, because `response$` emits responses (objects), not Observables (only `foo$$` emits `foo$`).